poutyne/framework/experiment.py

```diff
@@ -327,30 +555,41 @@ def load_best_checkpoint(self, *, verbose=False):
             print("Found best checkpoint at epoch: {}".format(best_epoch))
             print(metrics_str)
 
-        self.load_checkpoint(best_epoch)
+        self._load_epoch_checkpoint(best_epoch)
         return best_epoch_stats
 
-    def load_checkpoint(self, epoch):
-        ckpt_filename = self.best_checkpoint_filename.format(epoch=epoch)
-        self.model.load_weights(ckpt_filename)
-
-    def load_last_checkpoint(self):
+    def _load_last_checkpoint(self):
         self.model.load_weights(self.model_checkpoint_filename)
 
-    def test(self, test_loader, *, steps=None, load_best_checkpoint=True, load_last_checkpoint=False, seed=42):
-        if seed is not None:
-            # Make training deterministic.
-            random.seed(seed)
-            np.random.seed(seed)
-            torch.manual_seed(seed)
-
-        best_epoch_stats = None
-        if load_best_checkpoint:
-            best_epoch_stats = self.load_best_checkpoint(verbose=True)
-        elif load_last_checkpoint:
-            self.load_last_checkpoint()
-
-        test_loss, test_metrics = self.model.evaluate_generator(test_loader, steps=steps)
+    def test(self, test_generator, *, steps=None, checkpoint='best', seed=42):
+        """
+        Computes and returns the loss and the metrics of the attribute model on a given test examples
+        generator.
+
+        If the Experiment has logging enabled (i.e. self.logging is True), a :class:`~callbacks.CSVLogger` will save all
```

Comment:
It's not a logger that save those.

Reply:
Fixed by 26a15681204a829a0233458745d0de488912160d